File: sentimental-cash/cash.py
from cs50 import get_float

#declare variables
quarters,dimes,nickels,pennies,count = 0.25, 0.10, 0.05, 0.01, 0

#prompt and get user input
while True:
    change = get_float("Chage owed: ")
    if change > 0:
        break
#calculate the change
while change > 0:
    if change >= quarters:
        change =round(change-quarters,2)
        count+=1
    elif change >= dimes:
        change =round(change- dimes,2)
        count+=1
    elif change >= nickels:
        change = round(change- nickels,2)
        count+=1
    else:
        change =round(change- pennies,2)
        count+=1

#print user change
print(f" {count}")

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

//declare constant variables
#define QUARTERS 25
#define DIMES 10
#define NICKELS 5
#define PENNIES 1

int main(void)
{
    int nCoins = 0, change, quarters=0,dimes=0,nickels=0,pennies=0;


    //prompt & get user input
    do
    {
        change = get_int("Change owed:\t ");
    }
    while (change <= 0);
    printf("\n");

    //calculate coins to be dispensed
    while (change > 0)
    {
        if (change >= QUARTERS)
        {
            change -= QUARTERS;
            nCoins++;
        }
        else if (change >= DIMES)
        {
            change -= DIMES;
            nCoins++;
        }
        else if (change >= NICKELS)
        {
            change -= NICKELS;
            nCoins++;
        }
        else
        {
            change-=1;
            nCoins++;
        }
    }
    printf("Quarters%i\n", quarters);
    printf("Nickels%i\n", nickels);
    printf("Dimes: %i\n", dimes);
    printf("Pennies: %i\n", pennies);


}
